custom_components/tuya_local/devices/aygrochy_inline_wifi_air_vent.yaml

```diff
@@ -0,0 +1,298 @@
+name: Aygrochy fan
+primary_entity:
+  entity: climate
+  dps:
+    - id: 2
+      name: hvac_mode
+      type: string
+      mapping:
+        - dps_val: "ON"
+          value: "fan_only"
+        - dps_val: "OFF"
+          value: "off"
+        - dps_val: "TIMER"
+          value: "auto"
+          hidden: true
+        - dps_val: "AUTO"
+          value: "auto"
+        - dps_val: "ALARM"
+          value: "fan_only"
+          hidden: true
+    - id: 8
+      readonly: true
+      name: current_humidity
+      type: integer
+      range:
```

Comment:
This range is not needed (and would be wrong to go beyond 100 for %)

Reply:
fixed